Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -24,6 +24,12 @@ open class NavigationEventsManager {
     
     weak var dataSource: EventsManagerDataSource?
     
+    /**
+     Optional application metadata that that can help Mapbox more reliably diagnose problems that occur in the SDK.
+     For example, you can provide your application’s name and version, a unique identifier for the end user, and a session identifier.
+    */
+    var userInfo: [String: String?]? = nil
```

Comment:
I don't see this property is read anywhere. Can you please clarify what it is for and how it should be used? 

I could think that this is a set of properties to be included in each event. At the same time, user can provide another set of properties right to the method like `recordFeedback(type:description:userInfo:)` and these two sets will be merged. But right now it seems like it is not used at all.

Reply:
With newest commit, `userInfo` is set by the developer and then set as `appMetadata` for each event.